test/SlowTests/Utils/Analyzers/NGramAnalyzer.cs

```diff
@@ -4,25 +4,43 @@
 //  </copyright>
 // -----------------------------------------------------------------------
 
+using System;
 using System.IO;
 using Lucene.Net.Analysis;
 using Lucene.Net.Analysis.Standard;
-using Lucene.Net.Util;
 using Raven.Server.Documents.Indexes;
+using Raven.Server.Documents.Indexes.Static;
+using Version = Lucene.Net.Util.Version;
 
 namespace SlowTests.Utils.Analyzers
 {
     [NotForQuerying]
     public class NGramAnalyzer : Analyzer
```

Comment:
Am I missing something, or is this a test only analyzer?
Shouldn't we be moving this to the server?

Reply:
Yes, forgot to move it!